popup/js/search/fuzzySearch.js

```diff
@@ -34,48 +34,46 @@ export async function fuzzySearch(searchMode, searchTerm) {
 function fuzzySearchWithScoring(searchTerm, data) {
   /** Search results */
   const results = []
+  const haystack = data.map((el) => {
+    return el.searchString
+  })
 
-  // Score with fuzzysort is difficult to handle.
-  // We assume that the worst score we allow is -500.000
-  // and normalize score between this and 0
-  const scoreNormalizationFactor = -500000
-
-  const searchResults = fuzzysort.go(searchTerm, data, {
-    key: 'searchString',
-    limit: ext.opts.searchMaxResults,
-    // subtracting 1, otherwise the library will get confused by -0
-    threshold: ext.opts.searchFuzzyness * scoreNormalizationFactor - 1,
+  let uf = new uFuzzy({
+    // How many characters "in between" are allowed -> increased fuzzyness
+    intraIns: Math.round(ext.opts.searchFuzzyness * 4.2),
+    // When searchFuzzyness is greater than 0.7, also allow one
+    intraMode: ext.opts.searchFuzzyness >= 0.7 ? 1 : 0,
```

Comment:
if memory serves, this is mutually exclusive with `intraIns` > 1 (it basically lockes that setting to 1)

Reply:
also would appreciate your thoughts on https://github.com/leeoniya/uFuzzy/issues/11

Reply:
Ok, but that means it will just be locked to that level and since now intraMode errors are allowed it's more fuzzy anyway?

Reply:
right, though "more fuzzy" is doing a lot of lifting here. e.g. if you had intraIns: 3 (and got matches with 3 extra chars) then intraMode: 1 will be less fuzzy :). but generally intraMode: 1 is about same fuzz level as intraIns: 1 or 2